### DISCUSSION TITLE
How to enable ellipsis check?

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#7918] [Q&A]
Hello,

I want to enable an 'ellipsis' check, so that our translations will never have three dots when the source has an ellipsis. Under [component] > Settings > Translation > Enforced checks, I enabled the 'Ellipsis' check. To be sure I did it correctly, I clicked the question mark sign for [relevant documentation](https://docs.weblate.org/en/latest/admin/projects.html#component-enforced-checks). There I read the following:
> Enforcing the check does not automatically enable it, you still should enabled it using [Customizing behavior using flags](https://docs.weblate.org/en/latest/admin/checks.html#custom-checks) in [Translation flags](https://docs.weblate.org/en/latest/admin/projects.html#component-check-flags) or [Additional info on source strings](https://docs.weblate.org/en/latest/admin/translating.html#additional).

And so I went on to 'Customizing behavior using flags' in the documentation:
> You can fine-tune the Weblate behavior by using flags. This can be done […] in the [Component configuration](https://docs.weblate.org/en/latest/admin/projects.html#component) ([Translation flags](https://docs.weblate.org/en/latest/admin/projects.html#component-check-flags)).

Perfect, right what I need.

However, the check 'ellipsis' isn't listed there. Luckily I find, the following:

> Generally the rule is named `ignore-*` for any check, using its identifier, so you can use this even for your custom checks.

Great. I saw `ignore-ellipsis` in the list, so I can assume that 'ellipsis' is the check that I should add as translation flag. However, doing so creates an error:
![image](https://user-images.githubusercontent.com/11229646/180636931-f908c6bc-3eb6-44a2-803f-1f792e7d5495.png)

Did I interpret the documentation wrongly somewhere?

**Comment by nijel** [comment]
This check is enabled by default (what is the case for most of them).
